templates/service-template.yml

```
@@ -870,6 +885,9 @@ objects:
             - --kas-fleetshard-addon-id=${KAS_FLEETSHARD_ADDON_ID}
             - --cluster-logging-operator-addon-id=${CLUSTER_LOGGING_OPERATOR_ADDON_ID}
             - --alsologtostderr
+            - --reconciler-repeat-interval=${RECONCILER_REPEAT_INTERVAL}
+            - --leader-election-repeat-interval=${LEADER_ELECTION_RECONCILER_REPEAT_INTERVAL}
```

```suggestion
            - --leader-election-reconciler-repeat-interval=${LEADER_ELECTION_RECONCILER_REPEAT_INTERVAL}
```
the flag does not match the one defined in the ReconcilerConfig